<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>380</width>
    <height>374</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Anagram Finder</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <layout class="QGridLayout" name="gridLayout">
    <property name="leftMargin">
     <number>8</number>
    </property>
    <property name="topMargin">
     <number>8</number>
    </property>
    <property name="rightMargin">
     <number>8</number>
    </property>
    <property name="bottomMargin">
     <number>8</number>
    </property>
    <property name="spacing">
     <number>4</number>
    </property>
    <item row="0" column="0">
     <layout class="QVBoxLayout" name="verticalLayout">
      <property name="spacing">
       <number>8</number>
      </property>
      <item>
       <widget class="QLabel" name="label">
        <property name="text">
         <string>Type starting word</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLineEdit" name="lineEdit"/>
      </item>
     </layout>
    </item>
    <item row="1" column="0">
     <widget class="QTextBrowser" name="textBrowser"/>
    </item>
    <item row="2" column="0">
     <widget class="QProgressBar" name="progressBar">
      <property name="value">
       <number>0</number>
      </property>
      <property name="alignment">
       <set>Qt::AlignCenter</set>
      </property>
      <property name="textVisible">
       <bool>true</bool>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>380</width>
     <height>20</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuFile">
    <property name="title">
     <string>File</string>
    </property>
    <addaction name="separator"/>
    <addaction name="separator"/>
    <addaction name="actionExit"/>
   </widget>
   <widget class="QMenu" name="menuActions">
    <property name="title">
     <string>Actions</string>
    </property>
    <addaction name="actionSearch_Anagrams"/>
    <addaction name="actionFind_Longest"/>
    <addaction name="actionStop_Search"/>
   </widget>
   <addaction name="menuFile"/>
   <addaction name="menuActions"/>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
   <addaction name="actionLoad_Dictionary"/>
   <addaction name="actionSearch_Anagrams"/>
   <addaction name="actionFind_Longest"/>
   <addaction name="actionStop_Search"/>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
  <action name="actionLoad_Dictionary">
   <property name="icon">
    <iconset resource="resource.qrc">
     <normaloff>:/import</normaloff>:/import</iconset>
   </property>
   <property name="text">
    <string>Load Dictionary...</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+O</string>
   </property>
  </action>
  <action name="actionExit">
   <property name="text">
    <string>Exit</string>
   </property>
  </action>
  <action name="actionSearch_Anagrams">
   <property name="icon">
    <iconset resource="resource.qrc">
     <normaloff>:/lookup</normaloff>:/lookup</iconset>
   </property>
   <property name="text">
    <string>Search Anagrams</string>
   </property>
  </action>
  <action name="actionFind_Longest">
   <property name="icon">
    <iconset resource="resource.qrc">
     <normaloff>:/search</normaloff>:/search</iconset>
   </property>
   <property name="text">
    <string>Find Longest</string>
   </property>
  </action>
  <action name="actionStop_Search">
   <property name="icon">
    <iconset resource="resource.qrc">
     <normaloff>:/stop</normaloff>:/stop</iconset>
   </property>
   <property name="text">
    <string>Stop Search</string>
   </property>
  </action>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources>
  <include location="resource.qrc"/>
 </resources>
 <connections/>
</ui>
